### ISSUE TITLE
Another way to have different IPs?

### ISSUE THREAD

**Issue by Rast1234** [DeadlySurprise/FO2-Splitscreen#28]
Just a thought after reading a document about virtual network hacking: is it possible to specify address which game will bind to? If yes, it can be 127.0.0.**1**, 127.0.0.**2**, etc. Because all of them are link-local, but different. This could mitigate same-port issue.

**Comment by MorMund** [comment]
Yep, it should work. Good idea! Although I'd still like to eventually allow people to play with split screen in LAN or even online, which wouldn't be possible with this solution. But its definitely a vastly simpler and more robust solution!

**Comment by Rast1234** [comment]
Main host should bind to a LAN address+interface while others should see it from different link-local addresses. If it is possible to force game to bind to given interface and ip...

**Comment by MorMund** [comment]
FlatOut 2 is peer to peer, so every client has to be reachable. I don't see a way to make it work without a gateway. Though there might still be benefits to your solution even with the need for a gateway.
